tensornetwork/matrixproductstates/mps_class_test.py

```diff
@@ -208,3 +219,13 @@ def test_correlation_measurement(backend_dtype_values):
   actual[N // 2] = 0.25
   np.testing.assert_almost_equal(result_1, actual)
   np.testing.assert_allclose(result_2, np.ones(N) * 0.25)
+
+
+@pytest.mark.parametrize("dtype", [np.float64, np.complex128])
+def test_TMeigs(dtype):
+  D, d, N = 10, 2, 10
+  imps = InfiniteMPS.random(
+      d=[d] * N, D=[D] * (N + 1), dtype=dtype, backend='numpy')
+  eta, l = imps.transfer_matrix_eigs('r')
+  l2 = imps.unit_cell_transfer_operator('r', l)
+  np.testing.assert_allclose(eta * l.tensor, l2.tensor)
```

Comment:
Does this give 100% coverage to InfiteMPS? Testing seems kinda light right now.

Reply:
Most of the functionality of InfiniteMPS and FiniteMPS is derived from BaseMPS. I changed the tests to explicitly test all methods of BaseMPS. I'll add one more test for unit_cell_transfer_operator